src/apple/cli.rs

```diff
@@ -399,6 +399,7 @@ impl Exec for Input {
                 let macos_target = Target::macos();
 
                 let isysroot = format!("-isysroot {}", sdk_root.display());
+                let library_path = format!("{}/usr/lib", macos_sdk_root.display());
```

Comment:
Having this be part of the `macos_isysroot` expression (binding `(macos_isysroot, library_path)`) would prevent needing to expose `macos_sdk_root`; the (pre-existing) large number of bindings derived from each other can make this function difficult to reason about, so being aggressive about constraining that flow is warranted.